views/js/controller/admin/index.js

```diff
@@ -77,10 +77,13 @@ define([
     var _preventManagerRemoval = function(container){
         var $form = $(container).closest('form');
         var $submitter = $(':submit', $form);
+        if (!$submitter[0]) {
+            $submitter = $('button[title=Save]', $form);
+        }
 
         if (!_checkManagers($form)) {
             $submitter.addClass('disabled');
-            errorTooltip = tooltip.warning($submitter, errorMsgManagePermission, {
+            errorTooltip = tooltip.warning($form, errorMsgManagePermission, {
```

Comment:
maybe you need `$submitter` ?
```suggestion
            errorTooltip = tooltip.warning($submitter, errorMsgManagePermission, {
```